recipe/meta.yaml

```
@@ -114,13 +162,57 @@ outputs:
       license: BSD-3-Clause
       license_file: LICENSE
 
+  - name: clangxx_{{ cross_target_platform }}
+    script: install-clang++.sh
+    build:
+      skip: true  # [win or s390x or cross_target_platform == "linux-s390x"]
+      skip: true  # [compiler_vendor != "clang"]
+    requirements:
+      run:
+        # we need one fixed GCC version for clang builds; use last_symbol_bump
+        - libgcc-devel_{{ cross_target_platform }} {{ last_symbol_bump }}.*
+        # C++ compiler on linux needs a stdlib
+        - libstdcxx-devel_{{ cross_target_platform }}  {{ last_symbol_bump }}.*
+        # for activation of gcc env vars
+        - {{ pin_subpackage("clang_" ~ cross_target_platform, exact=True) }}
+        # for activation of binutils env vars
+        - {{ pin_subpackage("binutils_" ~ cross_target_platform, exact=True) }}
+        - sysroot_{{ cross_target_platform }}
+        - compiler-rt_{{ cross_target_platform }}
+        - compiler-rt =={{ compiler_version }}.*
```

Not sure if I'm missing something here, but I think we could leave out a bunch of requirements here, as they're all coming transitively through `clang_{{ cross_target_platform }}`.
```suggestion
        # C++ compiler on linux needs a stdlib
        - libstdcxx-devel_{{ cross_target_platform }}  {{ last_symbol_bump }}.*
        # for activation of gcc env vars
        - {{ pin_subpackage("clang_" ~ cross_target_platform, exact=True) }}
```